### ISSUE TITLE
SiteImprove: Headings should clearly describe the content they introduce

### ISSUE THREAD

**Issue by cory-skeers** [uiowa/uiowa#9211]
> Siteimprove AI is saying the name heading field for person content type is not descriptive and it’s categorized as AA.
[https://my2.siteimprove.com/Accessibility/998426/NextGen/Issue/1?ruleId=115&pageSegments=&issueKind=1&exceptTags=1,2&conformance=&siteTargetIssueKinds=1,2&wcagVersion=22&siteTarget.wcagVersion=22&siteTarget.conformanceLevels=0,1,3,4&siteTarget.issueKinds=1,2](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmy2.siteimprove.com%2FAccessibility%2F998426%2FNextGen%2FIssue%2F1%3FruleId%3D115%26pageSegments%3D%26issueKind%3D1%26exceptTags%3D1%2C2%26conformance%3D%26siteTargetIssueKinds%3D1%2C2%26wcagVersion%3D22%26siteTarget.wcagVersion%3D22%26siteTarget.conformanceLevels%3D0%2C1%2C3%2C4%26siteTarget.issueKinds%3D1%2C2&data=05%7C02%7Cits-web%40uiowa.edu%7C42dc7ce0b2724447a73108ddfaaf1aff%7C1bc445959aba4fc3b8ec7b94a5586fdc%7C1%7C0%7C638942352154412963%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=HFJ%2FcoFaaDO1nhiFpPpdAXSo09006331U0frJVFMNjM%3D&reserved=0)
Do you have any suggestions for how to fix this? I’m guessing others on campus are seeing similar issues, so maybe this would be a content type update? If you have suggestions for how to fix this with content updates, I’d love to know those.
Makur

> Our team is doing a good job making sure that webpages we create and manage are accessible and using Siteimprove to check. I am unsure how to solve the issue with headings for people.
Description
Headings should clearly describe the content they introduce. Siteimprove uses AI to review whether headings are descriptive. You can edit the review at any time if needed.

We need to investigate if this is a true issue or a false positive, and based on that answer, make an adjustment to the Person content type and/or have guidance available to provide to users of SiteImprove.

**Comment by quamsta** [comment]
Also seeing this in SiteImprove in reference to a person's profile page name. 

My opinion is that it's a false positive. A person's full name and credentials (if applicable) should be descriptive enough for a page's content heading `<h1>` to be sufficient. 

I think it would be very clunky and actually **hurt** accessibility and readability in practice to attempt to rename these headings something longer and more descriptive like "Profile for Jane Doe" or something.

**Comment by pyrello** [comment]
> Draft Siteimprove service page: https://its.uiowa.edu/services/siteimprove

This looks really good!

**Comment by briand44** [comment]
Now that the service page mentions how to get setup in SiteImprove, I think we should remove that info from https://webcommunity.sites.uiowa.edu/training. Possibly update the SiteImprove paragraph above it to link to the service page for more information.

**Comment by e-marie-w** [comment]
I've removed that last section and added a link in the paragraph above.
